Scripts/modelsystem.py

```diff
@@ -328,13 +296,18 @@ def run_iteration(self, previous_iter_impedance, iteration=None):
                 self.resultdata,
                 self.zdata_base.aggregations.municipality_mapping)
             self._calculate_noise_areas()
-            self._calculate_accessibility_and_savu_zones()
-            self.resultdata.print_line("\nMode shares", "result_summary")
-            for mode in mode_shares:
-                self.resultdata.print_line(
-                    "{}\t{:1.2%}".format(mode, mode_shares[mode]),
-                    "result_summary")
```

Comment:
I thought `results_summary.txt` to be just check if model run gives reasonable results and that it would be better to be visible in log. However, if result summary file is needed I can restore it.